### PR TITLE
Add IZ statement to holding item #1866

### PR DESCRIPTION
**Pull request by TobiasNx**
See #1866

### CODE REVIEW

**Comment by TobiasNx** [comment]
@dr0i this should be indexed so we can search for it.

**Comment by dr0i** [comment]
a) You want the `hasItem.inCollection.id` to be searchable I guess?
b) We should stick to one label. E.g. this is not good:
```json
    heldBy : {
      isil : DE-468,
      id : http://lobid.org/organisations/DE-468#!,
      label : Universitätsbibliothek Wuppertal
    },
    inCollection : [ { 
      id : http://lobid.org/organisations/DE-468#!,
      label : IZ of DE-468
    } ],
```
Also note the slight inconcruency by using  `heldBy.isil` while leaving this out for `inCollection`. I don't think this is crucial but maybe you just've missed it.

File: src/test/resources/alma-fix/990016782920206441.json [hunk 1]
```diff
@@ -158,6 +178,10 @@
       "id" : "http://lobid.org/organisations/DE-62#!",
       "label" : "Stadtbibliothek Wuppertal"
     },
+    "inCollection" : [ {
+      "id" : "http://lobid.org/organisations/7816#!",
```
Comment (dr0i) [inline src/test/resources/alma-fix/990016782920206441.json:182]: This is no valid organisation.

**Review by dr0i — CHANGES_REQUESTED** [review]

**Comment by TobiasNx** [comment]
Good catch this seems to be no true IZ but a bridge. We should be able to identify a bridge like 49HBZ_BRIDGE_FIZ .

https://service-wiki.hbz-nrw.de/pages/viewpage.action?pageId=949911658#NZPublishingf%C3%BCrDiscoveryAnbindung:Lobid-Members

File: src/test/resources/alma-fix/990016782920206441.json (hunk 1 above)
Comment (TobiasNx) [reply src/test/resources/alma-fix/990016782920206441.json]: I talked to @acka47 this is `Bibliotheken ohne zentrale Katalogisierung	49HBZ_BRIDGE_FIZ	7816` it has no ISIL an therefore no entry in lobid-organisations. I have introduced a dummy uri now: http://lobid.org/organisations/BRIDGE_FIZ-7816#! so we can identify this bridge.

**Comment by TobiasNx** [comment]
@dr0i yes please set it to searchable.

Thanks for the hint with the isil, but we do not need it here. The incollection statement is just to identify the IZ and not the organisation itself.

**Review by dr0i — APPROVED** [review]

**Pull request merged**